src/main/scala/org/ergoplatform/nodeView/wallet/ErgoWalletActor.scala

```diff
@@ -190,8 +190,8 @@ class ErgoWalletActor(settings: ErgoSettings) extends Actor with ScorexLogging {
         sender() ! BalancesSnapshot(height, registry.unconfirmedBalance, registry.unconfirmedAssetBalances)
       }
 
-    case ReadPublicKeys(from, until) =>
-      publicKeys.slice(from, until)
+    case r: ReadPublicKeys =>
```

Comment:
ReadPublicKeys(from, until) looks better to me here, but this is the matter of taste